handlers/views.go

```diff
@@ -90,6 +90,9 @@ func (s Server) guestLinkIndexGet() http.HandlerFunc {
 		"isActive": func(gl picoshare.GuestLink) bool {
 			return gl.IsActive()
 		},
+		"isDisabled": func(gl picoshare.GuestLink) bool {
```

Comment:
We don't need to do this, as the template can just call the struct's .IsDisabled function directly.

`isActive` set the wrong precedent, so I deleted that in #647.